gongt.go

```diff
@@ -505,6 +517,78 @@ func (n *NGT) Search(vec []float64, size int, epsilon float64) ([]SearchResult,
 	return result[:idx], nil
 }
 
+// StrictSearchbyParameter is C type stricted search function
+func StrictSearchbyParameter(vec []float64, ssp StrictSearchParameter) ([]StrictSearchResult, error) {
```

Comment:
by should be By

Reply:
thanks for review! 
replaced all "by" using "By"